espnet/espnet.c

```diff
@@ -299,39 +330,48 @@ static unsigned long _random(void) {
 	return 0;
 }
 // #if CONFIG_IDF_TARGET_ESP32S2 || CONFIG_IDF_TARGET_ESP32S3 || CONFIG_IDF_TARGET_ESP32C3
-//     uint32_t (* _slowclk_cal_get(void)
+//	 uint32_t (* _slowclk_cal_get(void)
 // #endif
 static void _log_write(uint32_t level, const char* tag, const char* format, ...) {
-	P(_log_write)
+	va_list argList;
+	printf("[%s] ", tag);
+	va_start(argList, format);
+	printf(format, argList);
```

Comment:
You should use `vprintf` here:
```suggestion
	vprintf(format, argList);
```

Output before this change:

```
[wifi] � (58720256) (null):
[wifi] static rx buf number 1070093076 is out of range
[wifi] L6
```

Output after this change:

```
[wifi] E (0) wifi:
[wifi] static rx buf number 0 is out of range
[wifi] 
```